app/views/layouts/partials/_form-password-field.blade.php

```diff
@@ -1,5 +1,5 @@
 <div class="field-group {{ setInvalidClass('password', $errors) }}">
   {{ Form::label('password', 'Password: ') }}
-  {{ Form::password('password') }}
+  {{ Form::password('password', array('placeholder' => 'Password must be at least six characters long')) }}
```

Comment:
Mostly since we should be/are using the literal syntax elsewhere in the code base.